### PR TITLE
[Keyboard] Adding support for OSAv2

### PR DESCRIPTION
**Pull request by blindassassin111**
Adding the OSAv2 MX PCB.

### CODE REVIEW

File: keyboards/viktus/osav2/keymaps/vial/rules.mk [hunk 1]
```diff
@@ -0,0 +1,8 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+VIAL_INSECURE = yes
```
Comment (lesshonor) [inline keyboards/viktus/osav2/keymaps/vial/rules.mk]:
```suggestion
```

File: keyboards/viktus/osav2/keymaps/vial/rules.mk [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+LTO_ENABLE = yes
+KEY_OVERRIDE_ENABLE = no
+VIALRGB_ENABLE = yes
```
Comment (lesshonor) [inline keyboards/viktus/osav2/keymaps/vial/rules.mk]:
```suggestion
```
`VIALRGB` is only for `RGB_MATRIX`.

**Review by lesshonor — CHANGES_REQUESTED** [review]
...You might want to set this as draft until the latest updates from upstream hit, since it doesn't compile as-is.

File: keyboards/viktus/osav2/keymaps/vial/rules.mk (hunk 1 above)
Comment (blindassassin111) [reply keyboards/viktus/osav2/keymaps/vial/rules.mk]: Oops. I thought I had removed this. Sorry.

**Comment by blindassassin111** [comment]
> ...You might want to set this as draft until the latest updates from upstream hit, since it doesn't compile as-is.

Done. Hopefully the wait isn't too long. How will I know when to come back and take it off draft?

**Comment by lesshonor** [comment]
> Done. Hopefully the wait isn't too long. How will I know when to come back and take it off draft?

Testing is happening [right now](https://github.com/vial-kb/vial-qmk/tree/merge-2023-03-12) but I don't have an ETA. I can just drop a comment here if you want...

**Comment by lesshonor** [comment]
You can un-draft this now, but you'll need to make some changes [given that `viktus/osav2` now exists in the `vial` branch](https://github.com/vial-kb/vial-qmk/tree/vial/keyboards/viktus/osav2).

**Pull request closed without merging**